Clarify ResourceQuota limit for PriorityClass (#26273)
kubernetes/website: content/en/docs/concepts/policy/resource-quotas.md

        values: ["cluster-services"]
```

> Then, create a corresponding resource quota object in `kube-system` namespace:

Reviewer: ```suggestion
Then, create a resource quota object in the `kube-system` namespace:
```

{{< codenew file="policy/priority-class-resourcequota.yaml" >}}

> ```shell script

Reviewer: should this be "shell" rather than "shell script"?

$ kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system
```

```
resourcequota/pods-cluster-services created
```

In this case, a pod creation will be allowed if:
1. Pod has no priority class and created in any namespace.
2. Pod has priority class other than `cluster-services` and created in any namespace.
3. Pod has priority class `cluster-services` and created in `kube-system` namespace, and passed resource quota check.

> Pod creation will be rejected if pod has priority class `cluster-services` and created in namespace other than `kube-system`.

Reviewer: ```suggestion

A Pod creation request is rejected if its `priorityClassName` is set to "`cluster-services`"
and it is to be created in a namespace other than `kube-system`.
```

        values: ["cluster-services"]
```

Then, create a corresponding resource quota object in `kube-system` namespace:

{{< codenew file="policy/priority-class-resourcequota.yaml" >}}

```shell script

> $ kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system

Reviewer 2: ```suggestion
$ kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system
```

Add the code black only for `kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system`
you can refer to line 323-325 of this chapter

Author: Done.

> resourcequota/pods-cluster-services created

Reviewer 2: ```suggestion
resourcequota/pods-cluster-services created
```
Add the code black only for `resourcequota/pods-cluster-services created`,
you can refer to line 327-331 of this chapter

Author: Done.

        values: ["cluster-services"]
```

Then, create a resource quota object in the `kube-system` namespace:

{{< codenew file="policy/priority-class-resourcequota.yaml" >}}

```shell

> $ kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system

Reviewer 3: nit:
```suggestion
kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system
```

```

```
resourcequota/pods-cluster-services created
```

In this case, a pod creation will be allowed if:

1.  the Pod's `priorityClassName` is not specified.
1.  the Pod's `priorityClassName` is specified to a value other than `cluster-services`.
1.  the Pod's `priorityClassName` is set to `cluster-services`, it is to be created
   in the `kube-system` namespace, and it has passed the resource quota check.

> A Pod creation request is rejected if its `priorityClassName` is set to `cluster-services`

Reviewer 3: nit: What rejects the request?

        values: ["cluster-services"]
```

Then, create a corresponding resource quota object in `kube-system` namespace:

```shell

> $ cat ./quota.yml

Reviewer 2: ```suggestion
$ cat ./quota.yml
```
Distinguish between 'command'and 'output'.

Author: Done.

- apiVersion: v1
  kind: ResourceQuota
  metadata:
    name: pods-cluster-services
  spec:
    scopeSelector:
      matchExpressions:
      - operator : In
        scopeName: PriorityClass
        values: ["cluster-services"]

> $ kubectl apply -f ./quota.yml -n kube-system

Reviewer 2: ```suggestion
$ kubectl apply -f ./quota.yml -n kube-system
```
Add this `kubectl` command output.

Author: Done.
